CMSgov/easi-app
# [EASI-3513] Easi 3513/admin note wrap up

EASI-3513

## Changes and Description

- Remove V1 Note code from UI + BE
- Removes `AddNoteV1` component and feature flag in UI `trbAdminNoteUpdates` - that feature flag will be removed in LaunchDarkly once this code is in prod



## How to test this change
1. Create a new TRB submission
2. Navigate to Admin Dashboard
3. Click into the created TRB
4. Click `Add a note` in the top right corner of the page and attempt to add a note
    - please try all types of notes
5. Confirm all note types are successfully created and are viewable in the `View notes` view



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
